src/index.js

```diff
@@ -75,29 +78,29 @@ app.use(passport.session());
 
 // Set up our passport strategy
 passport.use(new Strategy({
-    usernameField: 'email'
-  },
+  usernameField: 'email'
+},
```

Comment:
Not 100% sure about this indentation but I'll let it slide as it's a weird bit of code 😬